bundles/org.openhab.binding.volumio/src/main/java/org/openhab/binding/volumio/internal/VolumioHandler.java

```diff
@@ -257,6 +306,7 @@ private void bindDefaultListener() {
      */
     @Override
     public void initialize() {
+
```

Comment:
Very minor, but please remove empty line.
You could always check the SAT for this kind of issues.

Reply:
solved